src/gatling/simulations/scenarios/Solicitor_PRL_FL401_CaseCreate.scala

```diff
@@ -0,0 +1,1348 @@
+package scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import java.io.{BufferedWriter, FileWriter}
+import utils._
+
+object Solicitor_PRL_FL401_CaseCreate {
+
+  val BaseURL = Environment.baseURL
+  val MinThinkTime = Environment.minThinkTime
+  val MaxThinkTime = Environment.maxThinkTime
+
+  val CreatePrivateLawCase =
+
+    //set session variables
+    exec(_.setAll(
+      "ApplicantFirstName" -> (Common.randomString(5) + "appfirst"),
+      "ApplicantLastName" -> (Common.randomString(5) + "applast"),
+      "RespondentFirstName" -> (Common.randomString(5) + "respfirst"),
+      "RespondentLastName" -> (Common.randomString(5) + "resplast"),
+      "AppDobDay" -> Common.getDay(),
+      "AppDobMonth" -> Common.getMonth(),
+      "AppDobYear" -> Common.getDobYear(),
+      "RespDobDay" -> Common.getDay(),
+      "RespDobMonth" -> Common.getMonth(),
+      "RespDobYear" -> Common.getDobYear()))
+
+    /*======================================================================================
+    * Click the Create Case link
+    ======================================================================================*/
+
+    .group("XUI_PRL_FL401_030_CreateCase") {
+      exec(http("XUI_PRL_FL401_030_CreateCase")
+        .get(BaseURL + "/aggregated/caseworkers/:uid/jurisdictions?access=create")
+        .headers(Headers.xuiHeader)
+        .header("accept", "application/json")
+        .check(substring("PRIVATELAW")))
+
+      .exec(Common.userDetails)
+    }
+
+    .pause(MinThinkTime, MaxThinkTime)
+
+    /*======================================================================================
+    * Jurisdiction = Family Private Law; Case Type = C100 & FL401 Applications; Event = Solicitor Application
+    ======================================================================================*/
+
+    .group("XUI_PRL_FL401_040_SelectCaseType") {
+      exec(http("XUI_PRL_FL401_040_005_SelectCaseType")
+        .get(BaseURL + "/data/internal/case-types/PRLAPPS/event-triggers/solicitorCreate?ignore-warning=false")
+        .headers(Headers.xuiHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-start-case-trigger.v2+json;charset=UTF-8")
+        .check(jsonPath("$.event_token").saveAs("event_token"))
+        .check(jsonPath("$.id").is("solicitorCreate")))
+
+      .exec(Common.userDetails)
+
+      .exec(getCookieValue(CookieKey("XSRF-TOKEN").withDomain(BaseURL.replace("https://", "")).saveAs("XSRFToken")))
```

Comment:
This should be outside of the grouping, because it will contribute towards the trx time